.github/workflows/release-drafter.yml

```diff
@@ -22,10 +22,14 @@ jobs:
       - name: Build with Gradle
         run: |
           ./gradlew publishPublishMavenPublicationToLocalRepoRepository && tar -C build -cvf artifacts.tar.gz repository
+      - name: Build shadowJar
+        run:
+          ./gradlew shadowJar
       - name: Draft a release
         uses: softprops/action-gh-release@v1
         with:
           draft: true
           generate_release_notes: true
           files: |
             artifacts.tar.gz
+            build/libs/opensearch-sql-jdbc-shadow-*.jar
```

Comment:
Tha shado jar need to be bundled within `artifacts.tar.gz`. The jenkins server will only download that 1 artifact and unzip it. 
See how can you bundle all in one tar.

Thanks!

Reply:
Thanks for responding @gaiksaya, my worry is that including the shadowJar in the artifacts.tar is that Jenkins will also upload it to Maven which not ideal as per https://github.com/Bit-Quill/sql-jdbc/pull/8#issuecomment-1634975950.

What we need is:
1. have maven include non-shadowJar driver files and exclude shadowJar
2. OpenSearch downloads page to include shadowJar and non-shadowJar driver files.

Would you know if it is possible to exclude a file from artifacts.tar from getting published to Maven?

Reply:
Yes that is possible. What is published to maven is decided by these lines https://github.com/opensearch-project/sql-jdbc/pull/106/files#diff-619ac2b62b38c4694d7b69e89c001d78c5f853c5b39979c34d01012f778d26f0R12-R15

Right now if you unzip `artifacts.tar.gz`, these are the contents:
```
tree repository
repository
└── org
    └── opensearch
        └── driver
            └── opensearch-sql-jdbc
                ├── 1.4.0.0
                │   ├── opensearch-sql-jdbc-1.4.0.0-javadoc.jar
                │   ├── opensearch-sql-jdbc-1.4.0.0-javadoc.jar.md5
                │   ├── opensearch-sql-jdbc-1.4.0.0-javadoc.jar.sha1
                │   ├── opensearch-sql-jdbc-1.4.0.0-sources.jar
                │   ├── opensearch-sql-jdbc-1.4.0.0-sources.jar.md5
                │   ├── opensearch-sql-jdbc-1.4.0.0-sources.jar.sha1
                │   ├── opensearch-sql-jdbc-1.4.0.0.jar
                │   ├── opensearch-sql-jdbc-1.4.0.0.jar.md5
                │   ├── opensearch-sql-jdbc-1.4.0.0.jar.sha1
                │   ├── opensearch-sql-jdbc-1.4.0.0.pom
                │   ├── opensearch-sql-jdbc-1.4.0.0.pom.md5
                │   └── opensearch-sql-jdbc-1.4.0.0.pom.sha1
                ├── maven-metadata.xml
                ├── maven-metadata.xml.md5
                └── maven-metadata.xml.sha1
``` 

We can also zip the shadow jar outside `repository` folder. 

Question: Do we want all the below files or just the jar?
```
ll build/libs
total 12872
-rw-r--r--  1 gaiksaya  staff   261B Jul 26 11:08 opensearch-sql-jdbc-1.4.0.0-javadoc.jar
-rw-r--r--  1 gaiksaya  staff   109K Jul 26 11:08 opensearch-sql-jdbc-1.4.0.0-sources.jar
-rw-r--r--  1 gaiksaya  staff   192K Jul 26 11:08 opensearch-sql-jdbc-1.4.0.0.jar
-rw-r--r--  1 gaiksaya  staff   5.9M Jul 26 11:08 opensearch-sql-jdbc-shadow-1.4.0.0.jar
```